src/main/kotlin/org/rust/ide/formatter/RustLanguageCodeStyleSettingsProvider.kt

```diff
@@ -27,7 +27,10 @@ class RustLanguageCodeStyleSettingsProvider : LanguageCodeStyleSettingsProvider(
     override fun customizeSettings(consumer: CodeStyleSettingsCustomizable, settingsType: SettingsType) {
         when (settingsType) {
             BLANK_LINES_SETTINGS         -> {
-                // TODO Implement this.
+                consumer.showStandardOptions(
+                    "KEEP_LINE_BREAKS",
+                    "KEEP_BLANK_LINES_IN_DECLARATIONS",
+                    "KEEP_BLANK_LINES_IN_CODE")
```

Comment:
Looks like there is really no way to avoid string literals here... Am I correct that it will not throw an error if you make a typo here?